app/views/settings/_servers.html.erb

```diff
@@ -1,17 +1,22 @@
 <h2 class="govuk-heading-m">GovWifi servers</h2>
 
+<p class="govuk-hint govuk-!-margin-top-0 govuk-!-margin-bottom-0">
+  The
+  <%= link_to "offer GovWifi documentation", "https://docs.wifi.service.gov.uk/set_up/#step-3-create-a-new-wifi-installation", class: "govuk-link govuk-link--no-visited-state" %> explains how to connect your authenticators to these servers
```

Comment:
This line is really long, it would be easier to read in the code if it was broken across multiple lines.

Reply:
Odd that the linter doesn't pick this up. Should we change that?

Reply:
Potentially, although the linters are crude tools. People's judgement is often better.